packages/custom-storage/src/CustomStorage.ts

```diff
@@ -150,11 +160,11 @@ export class CustomStorage implements StorageRepository {
   }
 
   public uploadPublicDataset(dataset: Dataset, overwrite: boolean = false) {
-    return this.uploadDataset(dataset, this._publicSQLStorage, true, overwrite);
+    return this._uploadDataset(dataset, this._publicSQLStorage, true, overwrite);
```

Comment:
Are we going to send metrics headers when uploading datasets?

Reply:
We have already added a MetricsEvent to 'init' and 'createVisualization' (which includes uploading and linking its datasets) calls.

The idea here is to add headers for the most relevant actions. Is `uploadPublicDataset` directly used in Kepler? 

In fact, it would probably be pretty relevant to list here, **the use of all custom-storage public methods from the client**. The methods being called would be probably quite less than the ones available at the public interface of the class

Reply:
The current methods used in Kepler.gl are:
- `createVisualization` and `updateVisualization` from `CustomStorage` for saving maps
- `getVisualizations` and `getVisualization` from `CustomStorage` for loading maps
- `getVisualization` from `PublicStorageReader` that uses an own `SQL` class instance to load public maps

Reply:
Then those are the main requests I will add events on